Samples/DirectML_ESRGAN/main.cpp

```diff
@@ -0,0 +1,191 @@
+#pragma once
+
+#define UNICODE
+#define WIN32_LEAN_AND_MEAN
+#define NOMINMAX
+#define NODRAWTEXT
+#define NOGDI
+#define NOBITMAP
+#define NOMCX
+#define NOSERVICE
```

Comment:
should this be compiler flags?

Reply:
I don't have a strong preference. I can remove if truly desired, or move to compiler defs. Let me know if you feel strongly.